src/frontend/python/mod.rs

```diff
@@ -91,7 +91,7 @@ pub fn chiquito_ast_to_pil(witness_json: &str, rust_id: UUID, circuit_name: &str
 }
 
 fn add_assignment_generator_to_rust_id(
-    assignment_generator: AssignmentGenerator<Fr, ()>,
+    assignment_generator: AssignmentGenerator<Fr, NoTraceGenerator>,
```

Comment:
So here we should pass a Python front-end specific TraceGenerator type that gets the data the way it comes from the python front-end